ecs-init/version/version.go

```diff
@@ -0,0 +1,27 @@
+// This is an autogenerated file and should not be edited.
+
+// Copyright 2014-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// Package version contains constants to indicate the current version of the
+// ecs-init. It is autogenerated.
+package version
+
+// Version is the version of the ecs-init
+const Version = "1.13.0"
+
+// GitDirty indicates the cleanliness of the git repo when this ecs-init was built
+const GitDirty = true
+
+// GitShortHash is the short hash of this ecs-init build
+const GitShortHash = "c153a82"
```

Comment:
I think we missed copying this over in @richardpen's change set, but the `update-version.sh` script should set this to `UNKNOWN`.  Can you manually change it to `UNKNOWN` for now and then as a follow up create an issue to handle it?